src/libstd/thread/local.rs

```diff
@@ -334,6 +334,82 @@ impl<T: 'static> LocalKey<T> {
 }
 
 #[doc(hidden)]
+#[cfg(target_thread_local)]
+pub mod fast {
```

Comment:
Ah unfortunately the inner type is reexported as `#[doc(hidden)]` for "macro hygiene" and rustdoc doesn't see through that so still demands documentation on all the internals :(